content/client-lib-development-guide/versioning.textile

```diff
@@ -0,0 +1,17 @@
+---
+title: Ably Versioning
+section: client-lib-development-guide
+index: 19
+---
+
+Ably aims to follow the "semantic versioning scheme":http://semver.org/ for our "client library SDKs":https://www.ably.io/download, the "realtime protocol used by these SDKs":/client-lib-development-guide/protocol, our "REST API endpoints":/rest-api, and all other public endpoints, that support versioning, such as "Server-Sent Events":/sse.
```

Comment:
We (aim to) follow semver for public-facing APIs (which includes the REST api), but we don't really for the realtime protocol. E.g. connectionTimeserials will be a breaking protocol change, but we're doing that change in 1.3 rather than 2.0 because there's no change to the customer-facing client lib API. The policy we're effectively currently following for the realtime protocol is that breaking changes require a new spec version, which can be either a major or minor version.